lib/suspenders/app_builder.rb

```diff
@@ -342,6 +353,7 @@ def setup_stylesheets
 
     def install_refills
       generate "refills:import", "flashes"
+      run "rm app/views/refills/_flashes.html.erb"
```

Comment:
Do we need to call this here? It loos like the subsequent `remove_dir` would take care of this already.